ui/panels-plugin/src/plugins/line-chart/LineChartContainer.tsx

```diff
@@ -98,27 +97,20 @@ export function LineChartContainer(props: LineChartContainerProps) {
     );
   }
 
-  // TODO: toolbox functionality (ex. zoom, undo icons) will move to chart header
-  const toolboxOverrides: ToolboxComponentOption = {
-    orient: 'vertical',
-    top: 0,
-    right: 0,
+  const unit = props.unit ?? {
+    kind: 'Decimal',
+    decimal_places: 2,
   };
 
-  const gridOverrides: GridComponentOption = {
-    right: 40,
+  const legendOverrides = {
+    show: show_legend === true,
+    type: 'scroll',
+    bottom: 0,
   };
 
-  // enables zoom on drag without clicking 'Zoom' icon first
-  const dataZoomEnabled = false;
+  const gridOverrides: GridComponentOption = {
+    bottom: show_legend === true ? 35 : 0,
+  };
 
-  return (
-    <LineChart
-      height={height}
-      data={graphData}
-      grid={gridOverrides}
-      toolbox={toolboxOverrides}
-      dataZoomEnabled={dataZoomEnabled}
```

Comment:
Now dataZoom is on by default without having to click the icon and chart is restored on double click. The dataZoomEnabled prop was temporary until I could figure out that last part but isn't really needed anymore since I'm assuming users will always want to be able to zoom

https://user-images.githubusercontent.com/9369625/174319805-85f01f75-9b34-4847-847d-d5b4aa16fba8.mp4

Reply:
To make it a little more clear what's happening, I created a separate function for the dispatchAction lines that 'click' the invisible toolbox icon that enables dataZoom (see new [LineChart/utils.ts](https://github.com/perses/perses/blob/abf7b796542dc8a3dd33d8813ad86b1474f1fdfa/ui/components/src/LineChart/utils.ts) file).